src/mmw/mmw/settings/base.py

```diff
@@ -244,6 +244,7 @@ def get_env_setting(setting):
     'django.contrib.sessions.middleware.SessionMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
+    'apps.user.middleware.ItsiAuthenticationMiddleware',
```

Comment:
Is this the right place to put the new middleware? I also see that we have `mmw.middleware.BypassMiddleware`, and I could have added my `ItsiAuthenticationMiddleware` class there, but felt like this was more cohesive since all the other ITSI stuff is contained within `apps.user`.